uitools/import/Esri/ArcGISRuntime/Toolkit/FloorFilter.qml

```diff
@@ -0,0 +1,254 @@
+                        console.log(parent)
+                        internal.isFloorFilterCollapsed = false
+                    }
+                }
+            }
+
+            ToolBar {
+                Action {
+                    id: facility
+                    icon.source: "images/organization-24.svg"
+                    onTriggered: {
+                        facilityFilterMenu.visible = !facilityFilterMenu.visible
+                    }
+                }
+
+                ToolButton {
+                    Layout.fillWidth: true
+                    action: facility
+                    icon.color: "transparent"
+                }
+            }
+        }
+
+        GridLayout {
+            id: facilityFilterMenu
+            flow: GridLayout.TopToBottom
+            Layout.alignment: Qt.AlignBottom
+            rows: 4
+            //visible: internal.currentVisibileListView !== FloorFilter.VisibleListView.NONE
+            Rectangle {
+                Layout.fillHeight: true
+                Layout.rowSpan: 2
+                Layout.alignment: Qt.AlignHCenter
+                // if == facility-> gets set twice: once to 24 and after to 0.
+                width: internal.currentVisibileListView
+                       === FloorFilter.VisibleListView.FACILITY ? 24 : 0
+                border.color: "black"
+                Image {
+                    anchors.centerIn: parent
+                    id: leftChevronImg
+                    source: "images/chevron-left.svg"
+                    sourceSize.width: 24
+                    sourceSize.height: 24
+                    width: internal.currentVisibileListView
+                           === FloorFilter.VisibleListView.FACILITY ? sourceSize.width : 0
+                    height: internal.currentVisibileListView
+                            === FloorFilter.VisibleListView.FACILITY ? sourceSize.width : 0
+                    MouseArea {
+                        anchors.fill: parent
+                        onClicked: internal.currentVisibileListView
+                                   = FloorFilter.VisibleListView.SITE
+                    }
+                }
+            }
+
+            Image {
+                id: searchImg
+                sourceSize.width: 32
+                sourceSize.height: 32
+                source: "images/search.svg"
+            }
+
+            ListView {
+                Component.onCompleted: console.log("listview: ", count)
+                visible: true
+                Layout.columnSpan: 3
+                Layout.fillWidth: true
+                implicitHeight: contentHeight
+                implicitWidth: contentWidth
+                model: internal.currentVisibileListView
+                       === FloorFilter.VisibleListView.SITE ? controller.sites : controller.facilities
+                delegate: ItemDelegate {
+                    //width: parent.width
+                    text: '\u2022 ' + model.name
```

Comment:
🔧 Consider removing `'\u2022 '

The dot only appears on the highlighted item.

Note that `ItemDelegate` has a `highlighted: bool` toggle that can be set, that puts a checkmark next to the selected item. This was based on the behaviour of selects: https://esri.github.io/calcite-web/documentation/components/#selects

Reply:
Do you mean to entirely remove the dot and instead use the check mark? Or still keep it for an highlighted item?

Reply:
I would suggest using the checkmark for now. And the dot can become part of the style later, or replace the checkmark in the style.